### PR TITLE
test: cri: enable containerd debug logs

### PR DESCRIPTION
**Pull request by jcvenegas**
Debug logs are not enable when cri containerd tests are running.

Enable them for better CI debug.

Fixes: #4393

Signed-off-by: Carlos Venegas <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: integration/containerd/cri/integration-tests.sh
Comment (Jakob-Naucke) [inline integration/containerd/cri/integration-tests.sh:512]: @jcvenegas is this a WIP?

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

File: integration/containerd/cri/integration-tests.sh
Comment (jodh-intel) [inline integration/containerd/cri/integration-tests.sh:8]: Nit: This could be simplified to:

```suggestion
[ -n "${DEBUG}" ] && set -o xtrace
```

**Review by jodh-intel — APPROVED** [review]
Thanks @jcvenegas.

lgtm

**Review by GabyCT — APPROVED** [review]

**Pull request merged**
